src/hooks/use-run-tsx/index.tsx

```diff
@@ -90,6 +90,12 @@ export const useRunTsx = ({
       }
 
       const __tscircuit_require = (name: string) => {
+        if (
+          name === "./manual-edits.json" &&
+          preSuppliedImports["./manual-edits.json"] === ""
```

Comment:
we'll probably use null instead of empty string to this in the future

```suggestion
          !preSuppliedImports["./manual-edits.json"]
```